src/deepcode/lib/watchers/FilesWatcher.ts

```diff
@@ -0,0 +1,29 @@
+import * as vscode from 'vscode';
+import { ExtensionInterface } from '../../../interfaces/DeepCodeInterfaces';
+
+import { getGlobPatterns, ISupportedFiles } from '@deepcode/tsc';
+
+export default function createFileWatcher(
+  extension: ExtensionInterface,
```

Comment:
does extension have to be declared in both functions

Reply:
done